tests/test_environment.py

```diff
@@ -49,3 +57,21 @@ def test_resolve_user_dir_in_env_paths_env_not_set(monkeypatch):
     home = Path.home()
     env_dir = load_dir_from_environ("TEST_DIR", Path.home())
     assert env_dir == home
+
+
+def test_cli_global(monkeypatch, capsys):
+    if sys.platform.startswith("win"):
+        pytest.skip("This behavior is undefined on Windows")
+    assert not run_pipx_cli(["--global", "environment"])
+    captured = capsys.readouterr()
+    assert "PIPX_HOME=/opt/pipx" in captured.out
+    assert "PIPX_BIN_DIR=/usr/local/bin" in captured.out
+    assert "PIPX_MAN_DIR=/usr/local/share/man" in captured.out
+    assert "PIPX_SHARED_LIBS=/opt/pipx/shared" in captured.out
+    assert "PIPX_LOCAL_VENVS=/opt/pipx/venvs" in captured.out
+    assert "PIPX_LOG_DIR=/opt/pipx/logs" in captured.out
+    assert "PIPX_TRASH_DIR=/opt/pipx/.trash" in captured.out
+    assert "PIPX_VENV_CACHEDIR=/opt/pipx/.cache" in captured.out
+    # Checking just for the sake of completeness
+    assert "PIPX_DEFAULT_PYTHON" in captured.out
+    assert "USE_EMOJI" in captured.out
```

Comment:
I think it would be better to check for the whole output of the command here.

Reply:
Do you mean that I should have a one long string to compare output? This test is consistent with the test above which I think is a good thing: https://github.com/pypa/pipx/blob/0cfcdc1781d6bc366e7f72af5b3c59513d9ec118/tests/test_environment.py#L7-L20